Triangulation_2/include/CGAL/boost/graph/graph_traits_Delaunay_triangulation_2.h

```diff
@@ -53,7 +53,7 @@ namespace boost {
     typedef CGAL::detail::Edge<CGAL::Delaunay_triangulation_2<GT,TDS>, typename CGAL::Delaunay_triangulation_2<GT,TDS>::Edge>  edge_descriptor;
     typedef typename CGAL::Delaunay_triangulation_2<GT,TDS>::All_edges_iterator  edge_iterator;
 
-    typedef CGAL::detail::boost_all_vertices_iterator<Delaunay_triangulation> vertex_iterator;
+    typedef CGAL::Prevent_deref<typename Delaunay_triangulation::all_vertices_iterator> vertex_iterator;
```

Comment:
There is a red line in [CGAL-4.10-Ic-88]:
```
/mnt/testsuite/include/CGAL/boost/graph/graph_traits_Delaunay_triangulation_2.h:56:89: error: no type named 'all_vertices_iterator' in 'class CGAL::Delaunay_triangulation_2<CGAL::Epick, CGAL::Triangulation_data_structure_2<CGAL::Triangulation_vertex_base_with_id_2<CGAL::Epick>, CGAL::Triangulation_face_base_2<CGAL::Epick> > >'
     typedef CGAL::Prevent_deref<typename Delaunay_triangulation::all_vertices_iterator> vertex_iterator;
                                                                                         ^~~~~~~~~~~~~~~
```

[CGAL-4.10-Ic-88]: https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.10-Ic-88#BGL